build/deploy/metadata_base.libsonnet

```diff
@@ -43,4 +43,9 @@
       dest: error 'must specify destination email',
     },
   },
+  prometheus: {
+    external: false,
+    IP: '',  // Leaving blank means your cloud provider will assign an ephemeral IP
+    whitelist_ip_ranges: [],  // Empty list means firewall rules are open
```

Comment:
add a comment meaning anyone can access via public internet... in fact let's have an assertion somewhere that checks to see if we are making it external then this is not empty

Reply:
So we don't use this field at all unless, you want to expose prometheus externally, there is another service attached to the pod for intra cluster communication